docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.8"
+services:
+    # basic drupal site installation with dependencies and permissions configured
+    # use `docker exec -ti journal-cms_app_1 /bin/bash` to get a shell
+    app:
+        build: .
+        image: elifesciences/journal-cms #:${IMAGE_TAG}
+        volumes:
+            # todo: ensure writeable by www-data
+            - /var/log/journal-cms:/srv/journal/private/monolog
```

Comment:
If `/var/log/journal-cms` is supposed to be a route to a folder in the host machine I suggest that we have a folder within the project `./private/monolog` perhaps?

Reply:
go for it